pacs-integration-webapp/src/main/resources/db/changelog/liquibase.xml

```diff
@@ -182,4 +182,11 @@
             <column name="comment" type="varchar(255)"/>
         </addColumn>
     </changeSet>
+    <changeSet id="PACS-20220804" author="Mohankumar">
```

Comment:
also, will this work on on-premise install? the node name would be localhost or ipaddress, right?

Reply:
No Angshu. In an on-prem installation, this needs to be updated either with localhost or the ip of where DCM4CHEE (PACS) is running.